config/_default/config.toml

```diff
@@ -3,7 +3,7 @@ title = "Simon Désaulniers" # Title of your site
 theme = "zzo" # Name of Zzo theme folder in `themes/`.
 
 defaultContentLanguage = "fr" # Default language to use (if you setup multilingual)
-defaultContentLanguageInSubdir = true # baseURL/en/, baseURL/kr/ ...
+defaultContentLanguageInSubdir = false # baseURL/en/, baseURL/kr/ ...
```

Comment:
What is the incentive for turning this option off? Just want to understand this change. For now, I'm not sure which is best. I don't really have a preference so I'd be glad to include this change if I can understand why it's better.

If I understand correctly, I don't need to change my directory structure. Also, are the old links I've shared preserved as valid even though new links won't have `/fr/` in it?

Reply:
It is just my preference 😋
You don't need to change this param.
It just deletes default language folder directory